lib/overcommit/installer.rb

```diff
@@ -118,6 +125,32 @@ def can_replace_file?(file)
         overcommit_hook?(file)
     end
 
+    def preserve_old_hooks
+      return unless File.directory?(hooks_path)
+
+      ensure_directory(old_hooks_path)
+      Overcommit::Utils.supported_hook_types.each do |hook_type|
```

Comment:
That was my original thought, but I don't think we want to move hooks that aren't supported by Overcommit, and thus wouldn't be overwritten otherwise. So users get to keep their `prepare-commit-msg`, `pre-push`, _etc._ hooks without having to manually replace them from `.git/old-hooks`.